tests/integration/test_client.py

```diff
@@ -346,3 +366,56 @@ def test_workflow_find_by_type(client: AtlanClient):
     )
     assert results
     assert len(results) == 1
+
+
+def test_audit_find_by_user(
+    client: AtlanClient, current_user: UserMinimalResponse, audit_info: AuditInfo
+):
+    size = 10
+    assert current_user.username
+
+    results = client.audit.search(
+        AuditSearchRequest.by_user(current_user.username, size=size)
+    )
+    assert results.total_count > 0
+    assert size == len(results.current_page())
+    audit_entity = results.current_page()[0]
+    audit_info.qualified_name = audit_entity.entity_qualified_name
+    audit_info.guid = audit_entity.entity_id
+    audit_info.type_name = audit_entity.type_name
+
+
+@pytest.mark.order(after="test_audit_find_by_user")
+def test_audit_find_by_qualified_name(client: AtlanClient, audit_info: AuditInfo):
+    assert audit_info.qualified_name
+    assert audit_info.type_name
+    size = 10
+
+    results = client.audit.search(
+        AuditSearchRequest.by_qualified_name(
+            qualified_name=audit_info.qualified_name,
+            type_name=audit_info.type_name,
+            size=size,
+        )
+    )
+
+    assert results.total_count > 0
+    count = len(results.current_page())
+    assert count > 0 and count <= size
+
+
+@pytest.mark.order(after="test_audit_find_by_user")
+def test_audit_find_by_guid(client: AtlanClient, audit_info: AuditInfo):
```

Comment:
Would be awesome to see some deeper integration tests that flex parsing back the different kinds of audit history entries that could exist based on preceding tests (see `SQLAssetTest` in Java SDK as an example); rather than just checking there is any audit log entry at all (without introspecting whether its contents match expectations).

Reply:
I'm not sure that I'm following you. I see in the SQLAssetTest you are doing a searchByParentQN that seems to be looking for assets you created in the previous tests in the file. I'm not creating any assets. I'm just looking at whatever the most recent audit logs happen to be. Do you mean I should create an asset and then look for it by_guid and by_qualified_name?

Reply:
Never mind. I see you have a searchAuditLogByGuid test. I will look at that.

Reply:
Or add these integration tests into one of the other existing integration test modules, like `sql_asset_test.py` — that way we don't necessarily have to go through the process of creating even more connections, assets, tear-downs, etc (adding more execution time to the integration tests); instead we're just running these searches at the end of what's already there to confirm we get back what's expected.